### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ma-zah**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-# debugtopus  🐙💻
+# debugtopus  🐙 💻
 
-Tool to run your octomind tests against a local instance of your app.
+Hey! We use AI agent to auto-generate, host and run AI-generated end-to-end tests in standard Playwright code in the cloud. We work on both staging and production, as long as we can access the website.
```
Comment (ma-zah) [inline README.md:3]:
```suggestion
Hey!  Debugtopus is a tool to run Octomind tests against the local instance of your app. Octomind tests are AI auto-generated, hosted and run end-to-end tests in standard Playwright code in the cloud. They work on both staging and production, as long as we can access the website. 
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
-# debugtopus  🐙💻
+# debugtopus  🐙 💻
 
-Tool to run your octomind tests against a local instance of your app.
+Hey! We use AI agent to auto-generate, host and run AI-generated end-to-end tests in standard Playwright code in the cloud. We work on both staging and production, as long as we can access the website. 
+
+Whether you want to make sure you didn't break a new build or use the e2e tests for synthetic monitoring, we offer a fast and easy way to QA your web app. Here some [info](https://www.octomind.dev/?utm_source=github&utm_medium=debugtopus&utm_campaign=rm) on that and more in the [docs](https://www.octomind.dev/docs?utm_source=github&utm_medium=debugtopus&utm_campaign=rm).
```
Comment (ma-zah) [inline README.md:5]:
```suggestion
Here some [info](https://www.octomind.dev/?utm_source=github&utm_medium=debugtopus&utm_campaign=rm) on that and more in the [docs](https://www.octomind.dev/docs?utm_source=github&utm_medium=debugtopus&utm_campaign=rm).
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
+
+We created Debugtopus to help you pinpoint the bug if a test goes red. It's a tool to run Octomind tests against a local instance of your app.
```
Comment (ma-zah) [inline README.md:7]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,6 +1,9 @@
-# debugtopus  🐙💻
+# debugtopus  🐙 💻
+
+Hey!  Debugtopus is a tool to run Octomind tests against the local instance of your app. Octomind tests are AI auto-generated, hosted and run end-to-end tests in standard Playwright code in the cloud. They work on both staging and production, as long as we can access the website.
```
Comment (Germandrummer92) [inline README.md:3]:
```suggestion
Debugtopus is a tool to run Octomind tests against the local instance of your app. Octomind tests are AI auto-generated, hosted and run end-to-end tests in standard Playwright code in the cloud. 
```

File: README.md [hunk 5]
```diff
@@ -1,6 +1,9 @@
-# debugtopus  🐙💻
+# debugtopus  🐙 💻
+
+Hey!  Debugtopus is a tool to run Octomind tests against the local instance of your app. Octomind tests are AI auto-generated, hosted and run end-to-end tests in standard Playwright code in the cloud. They work on both staging and production, as long as we can access the website. 
+
+Here some [info](https://www.octomind.dev/?utm_source=github&utm_medium=debugtopus&utm_campaign=rm) on that and more in the [docs](https://www.octomind.dev/docs?utm_source=github&utm_medium=debugtopus&utm_campaign=rm).
```
Comment (Germandrummer92) [inline README.md:5]:
```suggestion
Here's some more [info](https://www.octomind.dev/?utm_source=github&utm_medium=debugtopus&utm_campaign=rm) on octomind and more in the [docs](https://www.octomind.dev/docs?utm_source=github&utm_medium=debugtopus&utm_campaign=rm).
```

Comment (Germandrummer92) [reply README.md:5]: Im not sure if UTM-links in readme are good style, devs are finicky like that. But also fine I guess if you need it.

**Review by Germandrummer92 — APPROVED** [review]

**Pull request merged**